sysbuild/Kconfig.zip

```diff
@@ -44,6 +44,46 @@ config DFU_ZIP_BLUETOOTH_MESH_METADATA
 	help
 	  Includes metadata for bluetooth mesh.
 
+if DFU_ZIP_BLUETOOTH_MESH_METADATA
+
+choice DFU_ZIP_BLUETOOTH_MESH_METADATA_FWID
+	prompt "Source of Firmware ID to include in metadata file"
+	optional
+
+config DFU_ZIP_BLUETOOTH_MESH_METADATA_FWID_CUSTOM
+	bool "Use a custom Firmware ID"
+	help
+	  The value to include as Firmware ID in the Bluetooth Mesh metadata file is set using
+	  ``SB_CONFIG_DFU_ZIP_BLUETOOTH_MESH_METADATA_FWID_CUSTOM_HEX``.
+	  Note: You are required to make sure that the Firmware ID used when initializing the DFU
+	  Server matches the ID in the metadata file.
```

Comment:
```suggestion
	  Note: You are required to make sure that the Firmware ID used when preparing the DFU
	  Server, for firmware distribution, matches the ID in the metadata file.
```

Reply:
This is more confusing, "for firmware distribution" makes me think of this somehow being related to a node in the Distributor role, but this note is about the DFU Server used as a target for the Fimrware Update, not distribution. Also, I like "initializing" better since you use the "_INIT" macro.